### PR TITLE
docs: various additions including metric definitions, FAQ, and headings

### PR DESCRIPTION
**Pull request by BigLep**
A better description of what changed is in the commit history (each commit was a pretty small change).

### CODE REVIEW

**Comment by BigLep** [comment]
I'm going to merge this PR since it is making things look better on my machine in prep for FDS 5.  Happy to incorporate feedback in followup PR.

File: README.md [hunk 1]
```diff
@@ -14,15 +14,24 @@ https://filecoin-project.github.io/filecoin-storage-providers-market/
...
 
 If you want to help with data presentation, layout, etc. please see [implementation details](#implementation-details) below and open a PR.
 
 Discussion and improvement proposals around service classes, SLOs, SLIs, and their defitions should occur in [filecoin-project/service-classes](https://github.com/filecoin-project/service-classes).
 
+## Development environment
+Below are the expected tools to install:
+* `npm` - Not needed if you're using [devcontainer.json](.devontainer/devcontainer.json)
+* [DuckDB CLI](https://duckdb.org/docs/installation/) - Not needed if you're using [devcontainer.json](.devontainer/devcontainer.json)
+* [Markdown All in One VS Code extension](https://markdown-all-in-one.github.io/) - This is used for markdown-all-in-one.github.io/docs/guide/table-of-contents.html, including its syntax for for ommitting certain headers.
```
Comment (davidgasquez) [inline README.md:30]: I can add this one to the Devcontainer too!

File: observablehq.config.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ export default {
     "parchment",
     "dashboard"
   ],
-  footer: false,
+  header: ({title, data, path}) => `<a href="/">🏠 Home</a> | <a href="/metrics">📈 Metrics</a> | <a href="/faq">❓ FAQ</a>`,
```
Comment (davidgasquez) [inline observablehq.config.js:24]: :pray: Simple and clean!

File: src/index.md [hunk 3]
```diff
@@ -218,11 +220,7 @@ Inputs.table(storage_providers, {
   rows: 20,
   select: false,
   format: {
-    provider_id: id => htl.html`<a href=/filecoin-storage-providers-market/provider/${id} target=_blank>${id}</a>`
+    provider_id: id => htl.html`<a href=./provider/${id} target=_blank>${id}</a>`
```
Comment (davidgasquez) [inline src/index.md:223]: I wasn't sure how to do this! :sweat_smile: I even tried making a variable that checked if it was running on `localhost`. The solution was much simpler, as it always happens.

**Review by davidgasquez — COMMENTED** [review]
Great PR. The site looks much more complete now.

**Pull request merged**